.github/workflows/beekeeper.yml

```diff
@@ -44,6 +44,8 @@ jobs:
           patch pkg/postage/postagecontract/contract.go .github/patches/postagecontract.patch
           patch pkg/postage/listener/listener.go .github/patches/listener.patch
           patch pkg/postage/service.go .github/patches/postageservice.patch
+          patch pkg/postage/batchstore/reserve.go .github/patches/postagereserve_gc.patch
```

Comment:
this needs to happen before the storage incentives setup is built/started. have a look how it was before we took the test out of the pipeline